src/compiler/build/build-ctx.ts

```diff
@@ -196,6 +196,11 @@ export class BuildContext implements d.BuildCtx {
   }
 }
 
+/**
+ * Generate a timestamp of the format `YYYY-MM-DDThh:mm:ss`, using the number of seconds that have elapsed since
+ * January 01, 1970, and the time this function was called
+ * @returns the generated timestamp
+ */
 export const getBuildTimestamp = () => {
```

Comment:
am I missing something, or does this produce the same output as `(new Date()).toISOString().split(".")[0]` ?

Reply:
Yeah, I think it does (create the same string)

Reply:
🤷‍♀️ seems to work so...idk! let's keep it